core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -26,28 +26,36 @@ import { ExplorationOpportunity } from '../opportunities-list-item/opportunities
 import constants from 'assets/constants';
 import { Subscription } from 'rxjs';
 
+type ExplorationOpportunitiesFetcherFunction = () => Promise<{
+  opportunitiesDicts: ExplorationOpportunity[];
+  more: boolean;
```

Comment:
Should we use a more appropriate property name here?

Reply:
Done